sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -30,8 +30,7 @@ import {
   KeyVaultClientGetKeysOptionalParams,
   KeyVaultClientRestoreKeyResponse,
   KeyVaultClientUpdateKeyResponse,
-  KnownApiVersion72Preview as KnownApiVersions,
-  ApiVersion72Preview as ApiVersion
+  KnownApiVersion72Preview as KnownApiVersions
```

Comment:
In this file, `KnownApiVersions` to figure out the default value for the api version to use. Can we use the constant `LATEST_API_VERSION` instead like before?